system-tests/README.adoc

```diff
@@ -0,0 +1,155 @@
+:toc: macro
+
+= tBTC v2 System tests
+
+This module contains the system test scenarios stressing the tBTC v2 system.
+
+toc::[]
+
+== Run system tests
+
+=== Prerequisites
+
+Please make sure you have the following prerequisites installed on your machine:
+
+- https://nodejs.org[Node.js] >14.17.4
+- https://yarnpkg.com[Yarn] >1.22.10
+
+=== Install dependencies
+
+To install dependencies, run:
+```
+yarn install
+```
+
+=== Bitcoin network requirements
+
+Test scenarios implemented in the `test` directory require
+a connection to the ElectrumX server synced with a Bitcoin
+network. Each test scenario can also define its own
+requirements related to the Bitcoin network. For example,
+deposit-related scenarios require a BTC account with a
+sufficient BTC balance. Please refer the documentation of
+specific test scenarios for more details.
+
+Theoretically, system tests should work on any Bitcoin network.
+However, we recommend to stick with the Bitcoin testnet as
+it is the easiest starting point.
+
+=== Ethereum network requirements
+
+All test scenarios require a connection to the Ethereum
+network with tBTC v2 contracts deployed on. Each of the
+test scenarios can also define a specific contract
+state required to make a successful pass. For example,
+deposit-related scenarios typically require a live wallet
+registered on the `Bridge` contract.
+
+System tests should work with any Ethereum network.
+Those network configurations should be placed in
+the link:hardhat.config.ts[Hardhat config file].
+However, the easiest way to run the system tests
+is using the Hardhat local network in the standalone mode.
```

Comment:
Yep we can do it in the next iteration since adding `geth` to the mix is much more complex and probably requires client implementation to be ready. I aimed to keep the system tests structure open for such extensions though.